management/locale/fr_FR/LC_MESSAGES/messages.po

```diff
@@ -435,15 +435,15 @@ msgstr "Ajouter une nouvelle catégorie"
 
 #: admin.php:182
 msgid "You don't have permission to access this page"
-msgstr "Vous n'avez pas les autorisations requises pour accéder à cette page"
+msgstr "Vous n'avez pas l'autorisation d’accéder à cette page"
```

Comment:
fixed in https://github.com/Coral-erm/Coral/pull/85/commits/5f9a9accf5d81520457c67c2dd3008931be83a83